sig-api-machinery/annual-report-2022.md

```diff
@@ -0,0 +1,126 @@
+# 2022 Annual Report: SIG API Machinery
+
+## Current initiatives
+
+1. What work did the SIG do this year that should be highlighted?
+
+   -
+   -
+   -
+
+2. What initiatives are you working on that aren't being tracked in KEPs?
+
+   -
+   -
+   -
+
+
+
+3. KEP work in 2022 (v1.26, v1.25, v1.24):
+  alpha:
+    - [3156 - HTTP3](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3156-http3) - v1.24
```

Comment:
The template needs to be updated, but this won't render right (view the file instead of the raw markdown).
It needs to either be another list item, or an additional space from the line above it (same for the beta and stable)

```
3. KEP work in 2022 (v1.26, v1.25, v1.24):

alpha:
- [3156 - HTTP3](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3156-http3) - v1.24
```

or 

```
3. KEP work in 2022 (v1.26, v1.25, v1.24):
- alpha:
  - [3156 - HTTP3](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/3156-http3) - v1.24
```